src/menu.rs

```diff
@@ -10,6 +10,7 @@ pub struct Menu {
     text: CloneCell<String>,
     fg: Color,
     entries: Vec<Box<Entry>>,
+    activated: CopyCell<bool>,
```

Comment:
Why not just use the original names. These are confusing.

Furthermore, why wrap it in a `Cell`?